text/0000-volta-run.md

```diff
@@ -59,14 +85,43 @@ There are alternative names we could use for the command, such as:
 
 However, `volta run` aligns with the "run script" command in both `npm` and `yarn`, so it seems to fit the best for this use-case.
 
+## Version Flags
```

Comment:
Thank you for adding this section. I do agree with the conclusion (that the more common case should be what we optimize), and I think that as written this RFC doesn't _prevent_ us from adding `--with <any valid volta install argument>` in the future as a follow up RFC.

👍

Reply:
Definitely not, we can always make changes in a follow-up, or if we decide after actually using the command that it needs to change.